packages/block-library/src/file/block.json

```diff
@@ -9,6 +9,12 @@
 		"href": {
 			"type": "string"
 		},
+		"fileId": {
```

Comment:
This change adds `fileId` as a block attribute, so that we can always identify the right file name to announce.